internal/config/load.go

```diff
@@ -79,26 +88,99 @@ func mergeAllExtends(fs afero.Fs, path string) (*viper.Viper, error) {
 		}
 	}
 
+	if err := remotes(fs, extends); err != nil {
+		return nil, err
+	}
+
```

Comment:
Originally, I hadn't added the option to specify the `config` to use, so it first searched for `lefthook.yml` inside of the remote and then `lefthook-local.yml`. Thinking on it now, I don't think this made sense even then since `lefthook-local.yml` should not be commited. 😅 I'll go ahead and remove it when moving this over to `internal/lefthook/install.go`.